src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
```

Comment:
The parameter 'days' could be more descriptive. Consider renaming it to 'numberOfDays' to make its purpose clearer.